deploy-tool/go.mod

```diff
@@ -1,17 +1,3 @@
 module github.com/abetterinternet/prio-server/deploy-tool
 
 go 1.15
-
-require (
```

Comment:
Wat

Reply:
dependabot seems to have (correctly) recognized that `deploy-tool` doesn't actually have any external dependencies: if you check its import statements it doesn't pull anything in except stdlib stuff. IIRC one of your in flight PRs moves deploy-tool to actually use the Kubernetes API so I think we should ignore these dependabot PRs for now.

Reply:
Oh... Okay that makes a lot more sense